.travis.yml

```diff
@@ -45,7 +45,7 @@ install:
 - unzip -qq gae-download.zip -d $TOOLS_DIR/google_appengine_1.9.19/
 - rm gae-download.zip
 
-script:
+script: --retry
```

Comment:
It could be better if there could be a maximum number of retry say 3, otherwise this will retry endlessly even if there is actual bugs in the code itself.
ie. `--retry 3`.
@bbriggs and @seanlip  what do you think?